src/test/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcherStateSpec.scala

```diff
@@ -3,14 +3,18 @@ package io.iohk.ethereum.blockchain.sync.regular
 import akka.actor.ActorSystem
 import akka.testkit.{TestKit, TestProbe}
 import io.iohk.ethereum.Mocks.MockValidatorsAlwaysSucceed
-import io.iohk.ethereum.BlockHelpers
+import io.iohk.ethereum.{BlockHelpers, WithActorSystemShutDown}
 import io.iohk.ethereum.network.PeerId
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpecLike
 
 import scala.collection.immutable.Queue
 
-class BlockFetcherStateSpec extends TestKit(ActorSystem()) with AnyWordSpecLike with Matchers {
+class BlockFetcherStateSpec
+    extends TestKit(ActorSystem())
```

Comment:
To consistency with other specs we should add here an actor system name - `BlockFetcherStateSpec_System`

Reply:
Good catch! Fixed: https://github.com/input-output-hk/mantis/pull/845/commits/58722643fbb62009c4d3b8400168830c888636c5